libs/blocks/quiz/quiz.js

```diff
@@ -22,6 +22,9 @@ const App = () => {
   const [currentStep, setCurrentStep] = useState(0);
   const [totalSteps, setTotalSteps] = useState(3);
   const [prevStepIndicator, setPrevStepIndicator] = useState([]);
+  const [nextQuizViewsExist, setNextQuizViewsExist] = useState(true);
+  const [urlParam, setUrlParam] = useState({});
+  const [btnAnalytics, setBtnAnalytics] = useState(null);
 
   useEffect(() => {
     (async () => {
```

Comment:
should be dataStrings and should we have comments for this function too?

Reply:
I have not added comments for all the functions knowingly. Just for the ones which might be a little heavy on logic and Wiki knowledge. Will change the variable names.